website/ocular/package.json

```diff
@@ -25,6 +25,7 @@
     "@luma.gl/core": "^7.0.0",
     "@luma.gl/webgl": "^7.0.0",
     "@probe.gl/stats-widget": "^3.0.1",
+    "gl": "^4.2.2",
```

Comment:
Hmm, this is not going to work on the server though... We'll need to see if this works when publishing.